swarm/swarm.go

```diff
@@ -41,6 +43,16 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/storage"
 )
 
+var (
+	startTime      time.Time
+	metricsTimeout = 5 * time.Second
+	startCounter   = metrics.NewCounter("stack,start")
+	stopCounter    = metrics.NewCounter("stack,stop")
```

Comment:
These should be `stack.start` and `stack.stop`.

Reply:
I think those are actually redundant. @holisticode what do you think?

Reply:
@nonsense in the sense that you already have metrics measuring start/stop events at process level? If that is so, then I agree it may be redundant.

Reply:
@holisticode we have the `uptime` gauge, and we also track docker daemon stats. We can infer from them how often swarm crashes for example.